app/controllers/users/applications_controller.rb

```diff
@@ -26,9 +29,11 @@ def new
     end
 
     def edit
+      authorize application
```

Comment:
It is the Rails Way, definitely.

In this case I thought it was clearer to put the method calls exactly they are needed, since about half the actions in that controller do not need the `before_action` (i.e. the whitelist is as long as the blacklist).

I will change for Rails Consistency though.